src/metadata/EntityMetadata.ts

```diff
@@ -442,7 +443,7 @@ export class EntityMetadata {
 
         // if target is set to a function (e.g. class) that can be created then create it
         if (this.target instanceof Function)
-            return new (<any> this.target)();
+            return ObjectInstantiator.createInstanceWithoutConstructor(<any> this.target);
```

Comment:
You know you could do something like:
```js
const object = {};
Reflect.setPrototypeOf(object, this.target.prototype);
return object
```

Reply:
What about `Object.create(this.target.prototype);` ? Looks more clear for me.

Reply:
@vlapo, yes `Object.create` will do the job, also `Reflect.construct`.
@XanderEmu not sure about `Reflect.setPrototypeOf`, see examples below

```js
class Post {
    constructor(title) {
        console.log("Inside constructor for " + title);
        this._title = title;
    }
}

let post1 = new Post("post 1"); // > 'Inside constructor for post 1'
console.log(post1 instanceof Post); // > true (OK !)
console.log(post1._title); // > 'post 1'  (OK !)

let post2 = Reflect.construct(function() {}, [], Post);
console.log(post2 instanceof Post); // > true  (OK !)
console.log(post2._title); // > undefined  (OK !)

const object = {};
let post3 = Reflect.setPrototypeOf(object, Post.prototype);
console.log(post3 instanceof Post); // > false  (NOT OK !)

let post4 = Object.create(Post.prototype);
console.log(post4 instanceof Post); // > true  (OK !)
console.log(post4._title); // > undefined (OK !)
```

Reply:
@XanderEmu Sorry, my bad, I messed it up, my example with `Reflect.setPrototypeOf` should have been coded properly:

```js
const object = {};
Reflect.setPrototypeOf(object, Post.prototype);
post3 = object;
console.log(post3 instanceof Post); // > true
```

All good then !